component/otelcol/exporter/prometheus/prometheus.go

```diff
@@ -80,6 +82,7 @@ func New(o component.Options, c Arguments) (*Component, error) {
 	converter := convert.New(o.Logger, fanout, convert.Options{
 		IncludeTargetInfo: true,
 		IncludeScopeInfo:  false,
+		AddMetricSuffixes: true,
```

Comment:
Shouldn't this be `c.AddMetricSuffixes` here?
This is something that we, as a team, should probably cover with better tests.

Reply:
Good catch!
Here the default value is set. And it is later overridden by a value from `c.AddMetricSuffixes` there:
```
if err := res.Update(c); err != nil {
	return nil, err
}
```
After your comment I realized that I forgot to update the method `Update` so it takes the value from `cfg. AddMetricSuffixes`

I actually am not quite sure why we set default values twice: once in `SetToDefault` method, and the second time in the `New` function. I'll write test to check config is unmarshalled correctly

Reply:
Right, the `Update` seems to be the correct place to use `c.AddMetricSuffixes`. Thanks for fixing.

In `New` we call `converter.New` and then immediately we'll call `converter.UpdateOptions` inside the `Update`, so what you pass in the `New` may not really matter. 

However, I think for readability of the code it's best we still make the values in both places to be the correct set of `convert.Options`. We can add a function that allows us to convert `Arguments` to `convert.Options` and use it in both places.

Reply:
sounds good! Did this